### PR TITLE
Keerthi | MOBN-2412 | Add. hideOtherInformationAndEmailAddress keyword to disable OtherInformation section

### PR DESCRIPTION
**Pull request by keerthi-magantii**
…d in patientCommonController to remove other information

### CODE REVIEW

File: ui/app/registration/views/patientcommon.html [hunk 1]
```diff
@@ -155,13 +155,13 @@
             <extra-patient-identifiers field-validation="::fieldValidation"></extra-patient-identifiers>
         </section>
 
-        <legend class="registraion_legend" ng-if="patientConfiguration.customAttributeRows().length > 0">
-            <span class="mylegend">{{ ::'REGISTRATION_LABEL_OTHER_INFO' | translate}}</span>
+        <legend ng-hide="hideOtherInformationAndEmailAddress" class="registraion_legend" ng-if="patientConfiguration.customAttributeRows().length > 0" >
+            <span ng-hide="hideOtherInformationAndEmailAddress" class="mylegend">{{ ::'REGISTRATION_LABEL_OTHER_INFO' | translate}}</span>
```
Comment (saisamyukthag) [inline ui/app/registration/views/patientcommon.html:159]: Do we need ng-hide for the child tag also?

File: .github/workflows/validate_pr.yml
Comment (saisamyukthag) [inline .github/workflows/validate_pr.yml:1]: Could you have a look at testcases for this file (Check if we need to add new ones?)

File: ui/app/registration/controllers/patientCommonController.js
Comment (saisamyukthag) [inline ui/app/registration/controllers/patientCommonController.js:1]: Could you have a look at testcases for this file (Check if we need to add new ones?)

File: ui/app/registration/views/patientcommon.html (hunk 1 above)
Comment (keerthi-magantii) [reply ui/app/registration/views/patientcommon.html:159]: fixed

File: ui/app/registration/controllers/patientCommonController.js
Comment (keerthi-magantii) [reply ui/app/registration/controllers/patientCommonController.js:1]: checked

**Review by saisamyukthag — APPROVED** [review]

**Pull request merged**